### DISCUSSION TITLE
how to create a new branch?

### DISCUSSION THREAD

**Discussion by ishan-pf** [Ayan-thecodeking/Here-Contributions-Begins#70] [General]
how can we create a new branch on github

**Comment by Ayan-thecodeking** [comment]
> how can we create a new branch on github

git branch <branch-name>
git checkout <branch-name> 
 git checkout -b <branch-name>

**Comment by Sri-vishnu-r1431** [comment]
We can also create a branch in github using the GUI provided by Github under main branch we can create a  new branch by giving a name and it will be created
